src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -145,38 +145,33 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
       });
     }
   }
-  async validateArweaveId(id: string): Promise<ValidationObject> {
-    const validatedIdObject = { name: '', status: false, error: '' };
-    validatedIdObject.name = 'Valid Arweave ID';
-    try {
-      new ArweaveTransactionID(id);
-      validatedIdObject.status = true;
-      return validatedIdObject;
-    } catch (error: any) {
-      validatedIdObject.status = false;
-      validatedIdObject.error = error.message ? error.message : error;
-      return validatedIdObject;
-    }
+  async validateArweaveId(id: string): Promise<ArweaveTransactionID> {
+    return new Promise((resolve, reject) => {
+      try {
+        const txid = new ArweaveTransactionID(id);
+        resolve(txid);
+      } catch (error: any) {
+        reject(error);
+      }
+    });
+  }
```

Comment:
nit: run lint/prettier to set proper spacing

Reply:
ran